fastlane/Fastfile

```diff
@@ -638,7 +638,7 @@ end
       readonly: options[:readonly] || is_ci,
       app_identifier: ["com.codality.NotationalFlow",
                        "com.codality.NotationalFlow.Share"],
-      template_name: "NotationalFlow Keychain Access",
+      template_name: "NotationalFlow Keychain Access (Distribution)",
```

Comment:
This is something that popped up while working on the automation to regenerate the provisioning profiles. With the previous value fail, the call to `match` failed.

On the Developer Portal, the Entitlement has "(Distribution)" in the name.